source/guide_keycloak.rst

```diff
@@ -0,0 +1,274 @@
+.. highlight:: console
+
+.. author:: Julius Künzel
+
+.. tag:: web
+.. tag:: sso
+
+.. spelling::
+    Keycloak
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/keycloak.svg
+      :align: center
+
+########
+Keycloak
+########
+
+.. tag_list::
+
+Keycloak is an open source identity and access management system which allows single sign-on for web applications and services.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`supervisord <daemons-supervisord>`
+  * :manual:`Backends <web-backends>`
+
+License
+=======
+
+Keycloak is released under the `Apache 2.0 license`_.
+
+Prerequisites
+=============
+
+Your URL needs to be setup:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+Download Keycloak
+-----------------
+
+Check the Keycloak_ website or `Github Repository`_ for the latest release and copy the download link to the .tar.gz file. Then use ``wget`` to download it. Replace the URL with the one you just got from GitHub.
+
+::
+
+ [isabell@stardust ~]$ wget https://github.com/keycloak/keycloak/releases/download/15.0.2/keycloak-15.0.2.tar.gz
+ […]
+ Saving to: ‘keycloak-15.0.2.tar.gz’
+
+ 100%[======================================>] 253,994,058 51,5MB/s   in 4,8s
+
+ 2021-11-19 22:36:18 (50,7 MB/s) - ‘keycloak-15.0.2.tar.gz’ saved [253994058/253994058]
+ [isabell@stardust ~]$
+
+Extract the archive
+-------------------
+
+Use ``tar`` to extract the archive and delete the archive. Replace the version in the archive file name with the one you just downloaded.
+
+::
+
+ [isabell@stardust ~]$ tar -xvzf keycloak-15.0.2.tar.gz
```

Comment:
This drops all the files in `~/keycloak-15.0.2`. What do you think about moving them to `~/keycloak`? That way all the paths for config, service file, etc. stay the same across updates. A `mv keycloak-15.0.2 keycloak` after the `tar` would do the job.

Reply:
Alternatively, add a symlink `~/keycloak-current` which always points to the current version.

Reply:
I decided for symlinks since the `mv` approach would make the update process more complex as far as I can see.